sqlalchemy_jsonapi/serializer.py

```diff
@@ -577,6 +577,7 @@ def delete_relationship(self, session, data, api_type, obj_id, rel_key):
                 if reverse_rel.direction == MANYTOONE:
                     permission = Permissions.EDIT
                 else:
+                    # pragma: no cover
```

Comment:
If we *can't* test it, we should leave it as uncovered code, not marking it up and away.

Sometimes it's not easy to follow all the code paths. I think it's telling that you couldn't figure out how to get it tested, and it makes me curious to know whether we ever even follow that code path.